src/Symfony/Component/Translation/MessageCatalogueInterface.php

```diff
@@ -82,23 +82,23 @@ public function defines($id, $domain = 'messages');
      *
      * @return string The message translation
      */
-    public function get($id, $domain = 'messages');
+    public function get($id, $domain = null);
 
     /**
      * Sets translations for a given domain.
      *
      * @param array  $messages An array of translations
      * @param string $domain   The domain name
      */
-    public function replace($messages, $domain = 'messages');
+    public function replace($messages, $domain = null);
 
     /**
      * Adds translations for a given domain.
      *
      * @param array  $messages An array of translations
      * @param string $domain   The domain name
```

Comment:
Note that the typehint of all those methods are now broken, should be `string|null`